components/webcompat_reporter/common/webcompat_reporter.mojom

```diff
@@ -26,8 +26,12 @@ struct ReportInfo {
   string? contact;
   array<ComponentInfo>? ad_block_components_version;
   array<uint8>? screenshot_png;
+  string? is_private_window;
```

Comment:
why it is a string `string?` instead of `bool?` ?

Reply:
and should that be an optional?

Reply:
I removed it completely